ngi_pipeline/engines/piper_ngi/database.py

```diff
@@ -63,6 +63,8 @@ def create_database_populate_schema(location):
     if not os.path.exists( os.path.dirname(location) ):
         try:
             os.makedirs( os.path.dirname(location)) 
+        except OperationalError:
```

Comment:
I had to look it up but try without except crashes the program.
The except is thrown when the directory can't be made (which includes when the directory already exists).
Running without a directory is fine. It's what ngi-pipeline has been capable of doing for quite a while now.